pkg/translator/prometheusremotewrite/otlp_to_openmetrics_metadata.go

```
@@ -66,9 +66,13 @@ func OtelMetricsToMetadata(md pmetric.Metrics, addMetricSuffixes bool, namespace
 			scopeMetrics := scopeMetricsSlice.At(j)
 			for k := 0; k < scopeMetrics.Metrics().Len(); k++ {
 				metric := scopeMetrics.Metrics().At(k)
+				metricName, err := metricNamer.Build(prom.TranslatorMetricFromOtelMetric(metric))
```

unfortunately I think this exported function will also need to return an error.